scheduler/graphql_mid/types.py

```diff
@@ -34,7 +34,8 @@ class SNightStats:
     def from_computed_night_stats(ns: NightStats) -> 'SNightStats':
         cf = json.dumps(ns.completion_fraction)
         pc = json.dumps(ns.program_completion)
-        return SNightStats(timeloss=ns.time_loss,
+        tl = json.dumps(ns.time_loss)
+        return SNightStats(time_loss=tl,
                            plan_score=ns.plan_score,
                            n_toos=ns.n_toos,
                            completion_fraction=cf,
```

Comment:
Other suggestion.
These `JSON` objects, can be send exactly as they are coming in the `ns`, graphql will handle them correctly and the UI will receive an object instead of a string that needs to be parsed into an object.
This method would be
```
def from_computed_night_stats(ns: NightStats) -> 'SNightStats':
    return SNightStats(time_loss=ns.time_loss,
                       plan_score=ns.plan_score,
                       n_toos=ns.n_toos,
                       completion_fraction=ns.completion_fraction,
                       program_completion=ns.program_completion)
```